### PR TITLE
Adjustments to the upcoming testnet configs

### PR DESCRIPTION
**Pull request by xgreenx**
Adjustments to the upcoming testnet configs:
- Decreased the max size of the contract/predicate/script to be 100KB.
- Decreased the max size of the transaction to be 110KB.
- Decreased the max number of storage slots to be 1760(110KB / 64).
- Removed fake coins from the genesis state.
- Renamed folders to be "testnet" and "dev-testnet".
- The name of the networks are "Upgradable Testnet" and "Upgradable Dev Testnet".

### CODE REVIEW

File: deployment/scripts/chainspec/testnet/state_config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "coins": [],
+  "messages": [],
+  "contracts": [
+    {
+      "contract_id": "7777777777777777777777777777777777777777777777777777777777777777",
+      "code": "9000000994318e6e453f30e85bf6088f7161d44e57b86a6af0c955d22b353f91b2465f5e6140000a504d00205d4d30001a4860004945048076440001240400005050c0043d51345024040000",
```
Comment (xgreenx) [inline deployment/scripts/chainspec/testnet/state_config.json:7]: The address of the fee recipient is the same for now and will be changed in a separate PR.

**Review by bvrooman — APPROVED** [review]

**Review by Theghost256 — APPROVED** [review]
